src/Plugin/Action/RemoveOgMembershipRole.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\Action;
+
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+
+/**
+ * Removes a role from a user.
+ *
+ * @Action(
+ *   id = "og_membership_remove_role_action",
+ *   label = @Translation("Remove a role from the selected members"),
+ *   type = "og_membership"
+ * )
+ */
+class RemoveOgMembershipRole extends ChangeOgMembershipRoleBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function execute(OgMembership $member = NULL) {
+    if (!$member) {
+      return;
+    }
+    $rid = $this->configuration['rid'];
+    $roles = $member->getRoles();
```

Comment:
Why not do something like:
`

    if (in_array($rid, $member->getRolesIds())) {
      $role = OgRole::load($rid);
      $member->revokeRole($role);
      $member->save();
    }
`